eclass/coreos-kernel.eclass

```diff
@@ -20,8 +20,26 @@ SLOT="0/${PVR}"
 SRC_URI=""
 IUSE="audit selinux"
 
-DEPEND="=sys-kernel/coreos-sources-${COREOS_SOURCE_VERSION}
-	sys-kernel/bootengine:="
+DEPEND="
+	app-arch/gzip
+	app-shells/bash
+	sys-apps/coreutils
+	sys-apps/findutils
+	sys-apps/grep
+	sys-apps/ignition:=
+	sys-apps/less
+	sys-apps/sed
+	sys-apps/shadow
+	sys-apps/systemd
+	sys-apps/util-linux
+	sys-fs/btrfs-progs
```

Comment:
i suppose btrfs-progs would pull in sys-fs/e2fsprogs, but it might be better to be explicit about it